pydantic/utils.py

```diff
@@ -1,4 +1,6 @@
 import warnings
+from inspect import Parameter, Signature, signature
```

Comment:
please remove this from top level imports.

* `Signature` is only required as a type hint so can go into `if TYPE_CHECKING:`
* `Parameter` and `signature` can be moved into `generate_model_signature`

Reply:
But It used each time when model is created. Why put relatively slow import in a function that gonna be called so many times instead of importing it just once here? That way it's gonna impact on dynamic model creation.

Reply:
no, if you repeat an import which is already loaded, it's cached.

Also model's aren't immediately created every time pydantic is imported.

We could decide that inspect is used so commonly that we just accept it will always be imported by pydantic, but that's another discussion

Reply:
Ok, I get it. Will fix soon